components/proxy/src/main/java/com/hotels/styx/admin/AdminServerBuilder.java

```diff
@@ -100,18 +104,75 @@ public AdminServerBuilder backendServicesRegistry(Registry<BackendService> backe
         return this;
     }
 
+    private static List<Route> routesForPlugin(NamedPlugin namedPlugin) {
```

Comment:
Looks like some methods have been accidentally rearranged here, making it hard to read the diff and giving the false impression that methods have been deleted and new ones created.